src/renderer/controllers/torrent-list-controller.js

```diff
@@ -40,6 +40,11 @@ module.exports = class TorrentListController {
 
   // Shows the Create Torrent page with options to seed a given file or folder
   showCreateTorrent (files) {
+    // You can only create torrents from the home screen.
+    if (this.state.location.url() !== 'home') {
+      return dispatch('error', 'Please go back to the torrent list before creating a new torrent.')
```

Comment:
The one other place I think it can be called is via the menu item "Create New Torrent..." when the player is open. Shouldn't the player also do `ipcRenderer.send('setAllowNav', false)`? This seems like the right place to handle it. Then this line can be removed here.